cpp/doAll_Zp.C

```diff
@@ -1,6 +1,7 @@
 {
   gROOT->ProcessLine(".L ../NanoCORE/NANO_CORE.so");  // NanoCORE library
   gROOT->ProcessLine(".L ScanChain_Zp.C+");  // Macro that performs the selection
+  //gROOT->ProcessLine(".L ScanChain_Zp_C.so");  // Macro that performs the selection, in case already compiled
```

Comment:
Do we need this line? I think that the previous line does not recompile if there are no changes.

Reply:
I left it there, since it helps reducing overhead time in case there's nothing to compile (based on my experience)